core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -142,8 +156,11 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
      * A map storing current activations based on ActivationId.
      * The promise value represents the obligation of writing the answer back.
      */
-    case class ActivationEntry(created: Instant, promise: Promise[WhiskActivation])
-    private val activationMap = new TrieMap[ActivationId, ActivationEntry]
+    case class ActivationEntry(id: ActivationId, subject: String, invokerIndex: Int, created: Instant, promise: Promise[WhiskActivation])
```

Comment:
should be `Identity` vs `Subject`? is this limit intended for subject or namespace is the operative question.